jbmc/src/java_bytecode/remove_java_new.cpp

```diff
@@ -86,12 +86,12 @@ goto_programt::targett remove_java_newt::lower_java_new(
   typet object_type = rhs.type().subtype();
 
   // build size expression
-  exprt object_size = size_of_expr(object_type, ns);
-  CHECK_RETURN(object_size.is_not_nil());
+  const auto object_size = size_of_expr(object_type, ns);
+  CHECK_RETURN(object_size.has_value());
 
   // we produce a malloc side-effect, which stays
   side_effect_exprt malloc_expr(ID_allocate, rhs.type(), location);
-  malloc_expr.copy_to_operands(object_size);
+  malloc_expr.copy_to_operands(object_size.value());
```

Comment:
No need to change these, but for future reference you can use `operator*` for brevity

Reply:
sure